plugins/analysis/dashboard/fpc_storage.go

```diff
@@ -23,6 +24,8 @@ type FPCRecord struct {
 	Opinions []int32 `json:"opinions" bson:"opinions"`
 	// Outcome defines final opinion of the conflict.
 	Outcome int32 `json:"outcome" bson:"outcome"`
+	// DateTime defines the datetime when the conflict has been finalized.
```

Comment:
`DateTime` is a terrible name. Can we call it just time or FinalizeTime or something...